[login] new plugin based in last

mikelolasagasti opened the pull request:
login.defs is required by some security compliance checks and could
be useful to collect it. Added to system plugin, as seems the most
neutral plugin to collect it.

Closes #1808

Signed-off-by: Mikel Olasagasti Uranga <mikel@olasagasti.info>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

bmr-cymru commented:
I agree we should collect this, but I don't think the `system` plugin is the right place for it - since we cleaned that plugin up a few years ago it's intended to be just low-level common systemwide stuff: primarily, the content of `/proc/sys`,`glibc` configuration and the two common configuration paths `/etc/default` and `/etc/sysconfig`.

I think we'd want whatever plugin collects `login.defs` to be included in the `identity` profile, which `system` isn't a part of - and the other data it collects doesn't really seem to fit in that profile.

It might be best to merge this with the `last` plugin (and add that to the `identity` profile, which it should be a part of anyway), to create a new `login` plugin. The files are provided by the same package on distributions that use `shadow-utils` so there's a good fit in terms of package triggers.

mikelolasagasti commented:
Thanks for the review @bmr-cymru, I was not aware of the cleaning done for `system` plugin. I moved `last` to `login`, added `identity` profile and added files `logins.defs`and `default/useradd` files from `shadow-utils`.
